src/main/scala/com/databricks/labs/overwatch/pipeline/BronzeTransforms.scala

```diff
@@ -949,6 +949,24 @@ trait BronzeTransforms extends SparkSessionWrapper {
 
   }
 
+  /**
+   *
+   * @param cloudProvider
+   * @param inputCol
+   * @param isMultiWorkSpaceDeployment
+   * @param organisationId
+   * @return
+   */
+  private def fetchClusterLogConfiguration(cloudProvider: String, inputCol: Column,
+                                   isMultiWorkSpaceDeployment: Boolean, organisationId: String): Column = {
+    if(cloudProvider.toLowerCase() == "gcp" && isMultiWorkSpaceDeployment)
```

Comment:
shouldn't we only do this for GCP REMOTE workspaces? GCP local master / driving workspaces should just use the log path in the cluster config, no?

Reply:
yes, we are doing this only for GCP multiworkspace

Reply:
add where local workspace orgID != currentOrgId (current from config)

Reply:
done